winui/Kanban/Getting-started.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+# Getting Started with WinUI Kanban (SfKanban)
+
+This section provides an overview of working with WinUI Kanban and a step-by-step guide to configuring and using it effectively in your application.
+
+## Creating an application with WinUI Kanban
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).
+
+2. Add reference to the `Syncfusion.Kanban.WinUI` NuGet.
+
+3. Import the control namespace `Syncfusion.UI.Xaml.Kanban` in XAML or C# code.
+
+4. Initialize the WinUI Kanban control.
+
+{% capture codesnippet1 %}
+{% tabs %}
+
+{% highlight xaml tabtitle="MainWindow.xaml" hl_lines="7 12" %}
+<Window x:Class="GettingStarted.MainWindow"
```

Comment:
@Kerubananthan3761 , follow this standards

![image](https://github.com/user-attachments/assets/0fb5b0ed-acc3-4df8-bf4d-89351bce1d07)

Reply:
Example codes changed